code/go/0chain.net/smartcontract/storagesc/allocation.go

```diff
@@ -1717,6 +1722,23 @@ func (sc *StorageSmartContract) finishAllocation(
 
 	alloc.Finalized = true
 
+	for _, challenge := range challenges.OpenChallenges {
+		ba, ok := alloc.BlobberAllocsMap[challenge.BlobberID]
+
+		if ok {
+			ba.Stats.FailedChallenges++
+			ba.Stats.OpenChallenges--
+			alloc.Stats.FailedChallenges++
+			alloc.Stats.OpenChallenges--
```

Comment:
Update also totalChallenges

Reply:
We are not updating total challenges as we are not using deltas anymore. It is in sync with MPT already. I have changes failed challenges with successful challenges